helpers/utils/mail-sender.js

```diff
@@ -0,0 +1,35 @@
+import nodemailer from 'nodemailer';
+import dotenv from 'dotenv';
+import sgTransport from 'nodemailer-sendgrid-transport';
+
+dotenv.load();
+const options = {
+  auth: {
+    api_key: process.env.SENDGRID_API_KEY,
+  }
+};
+const client = nodemailer.createTransport(sgTransport(options));
+const mailSender = {
+  send(infos) {
+    const sendEmailPromise = new Promise((resolve, reject) => {
+      // Prepare the link
+      const link = `${process.env.APP_LINK}/v1/auth/verify?t=${infos.token}`;
+      const email = {
+        from: process.env.APP_EMAIL,
+        to: infos.email,
+        subject: 'Email verification from RidersAhBackend',
+        text: `<html>Please click on the following <strong><a href="${link}"> link</a></strong> to verify your email</html>`,
```

Comment:
you can remove the text key from email object as it is the same as html key. for reusability of this function you can also pass as argument the subject and html of the email as you did for the to key with infos.email. you can also use infos.subject and infos.html on the two keys.

Reply:
I have done it